replica/replica.go

```diff
@@ -243,6 +248,32 @@ func IsHeadDisk(diskName string) bool {
 	return false
 }
 
+func (r *Replica) setUUID() string {
```

Comment:
can we rename it as readUUID() ?

Reply:
setUUID is creating a new uuid and writing to disk. If it is already there, it'll be read and r.UUID will be set.
We don't have a dedicated method to read UUID